gradle.properties

```diff
@@ -18,4 +18,6 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+
+VERSION=1.1.4
```

Comment:
let's keep track of this version in a separate `version.properties` file. `gradle.properties` should be used for gradle related build properties.

Reply:
fair enough